jans-cli-tui/cli_tui/plugins/010_auth_server/ssa.py

```
@@ -140,7 +140,7 @@ def do_delete_ssa_template(cdialog):
         self.app.show_jans_dialog(templatesDialog)
 
 
-    def update_ssa_container(self, start_index=0, search_str=''):
+    def update_ssa_container(self, start_index=0, search_str='', display_no_matching=True):
```

Ruff (0.15.21)

[warning] Boolean default positional argument in function definition

(FBT002)